elasticsearch/kibana/map.jinja

```diff
@@ -3,15 +3,18 @@
         'config': {
             'elasticsearch_url': 'http://localhost:9200',
             'server.host': '127.0.0.1',
-            'server.port': '5601'
+            'server.port': 5601
         },
         'nginx_config': {
             'cert_path': '/etc/salt/ssl/certs/kibana.yourdomain.com.crt',
             'key_path': '/etc/salt/ssl/certs/kibana.yourdomain.com.key',
         },
         'nginx_extra_config_template_source': 'salt://elasticsearch/kibana/templates/nginx_extra.conf',
         'ssl_directory': '/etc/salt/ssl/certs',
-        'es_client_node': True
+        'es_client_node': True,
+        'kibana_env': [
+            'NODE_OPTIONS=--max-old-space-size=1024'
+        ]
```

Comment:
Question: Is the explanation in this comment an accurate explanation as to why this works? https://github.com/elastic/kibana/issues/5170#issuecomment-157655647

It seems reasonable to me, but I just wanted to make sure.